Resources/Locale/en-US/guidebook/chemistry/effects.ftl

```diff
@@ -267,11 +267,11 @@ reagent-effect-guidebook-make-sentient =
         *[other] make
     } the metabolizer sentient
 
-reagent-effect-guidebook-make-tree =
+reagent-effect-guidebook-make-polymorph =
     { $chance ->
-        [1] Makes
-        *[other] Make
-    } the metabolizer a tree
+        [1] Polymorphs
+        *[other] polymorph
+    } the metabolizer
 
```

Comment:
Into what?

Reply:
Do I just put "into a tree" back?
Failing to see how would I pass a variable there telling it into what it would polymorph without hardcoding it.

Reply:
Pass the name of the prototype as a fluent argument, or similar.

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/109166122/d8346665-414d-4ce2-bdf7-7f99ca9c23fb)
aye

Reply:
it needs a localized name not just the prototype id @Equivocateur